openmetadata-ui/src/main/resources/ui/src/components/DataInsight/DataInsightChartCard.tsx

```diff
@@ -372,7 +538,10 @@ export const DataInsightChartCard = ({
                           activeKeys.length ? activeKeys.includes(entity) : true
                         }
                         label={`${round(latestData[entity] ?? 0, 2)}${
-                          isPercentageGraph ? '%' : ''
+                          isPercentageGraph &&
+                          type !== SystemChartType.TotalDataAssetsByTier
+                            ? '%'
+                            : ''
```

Comment:
Although the Summary for Tier is a Percentage, the Numbers on the chart are absolute values.
Added this condition to fix it for now.

Should be improved!